index.html

```diff
@@ -22,58 +23,46 @@
+                        <a class="nav-link active" aria-current="page"href="contacts.html">Contact</a>
                     </li>
                 </ul>
             </div>
         </div>
     </nav>
-
-    <div class="container mt-3">
-        <h1>Bienvenue sur ma page Bootstrap</h1>
-        <p>Cette page est créée avec Bootstrap.</p>
-    </div>
-    <div class="container">
-        <div class="row">
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/dog.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
-                    </div>
-                </div>
-            </div>
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/woman.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
-                    </div>
-                </div>
-            </div>
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/man.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
-                    </div>
-                </div>
-            </div>
+           <div class="containermt-3">
+              <h1> HOME</h1>
+              <P> Soyez les bienvenues </P>
+           </div>
+           <div class="container">
+               <div class="row">
+                   <div class="col-12">
+                        <img src="./assets/image1.jpg" class="image1" alt="image1">
+                         <p class="text">Je suis [Toufik Dellys], et je suis ravi de vous accueillir sur mon espace. Je suis un developpeur web, et j'ai une passion pour tous ce qui est desingne.
+        
+                           Au fil des ans, j'ai eu la chance de travailler avec de nombreux clients et de collaborer avec d'autres professionnels pour fournir des solutions sur mesure pour chaque situation. J'ai une grande expérience dans le front-End, et je suis fier de mes résultats.
```

Comment:
If you want to separate it in multiple paragraphs, instead to break the line, you need to use multiple `<p>` for each paragraph.